core/eolearn/tests/test_eodata.py

```diff
@@ -172,6 +172,41 @@ def test_get_feature(self):
 
         self.assertTrue(np.array_equal(eop_bands, bands), msg="Data numpy array not returned properly")
 
+    def test_shallow_copy(self):
+        path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../../example_data/TestEOPatch')
+        eop = EOPatch.load(path)
+
+        eop_copy = eop.copy()
+        assert eop == eop_copy
+        assert eop is not eop_copy
+
+        eop_copy.mask['CLM'] += 1
+        assert eop == eop_copy
```

Comment:
Maybe you can also compare id's of objects to make sure it's shallow?

Reply:
The line `assert eop is not eop_copy` is meant to compare ids. Do you think it would be nicer to do `id(eop) != id(eop_copy)`?

Reply:
I would like the comparison on feature level, e.g. `eop.mask['CLM'] is eop_copy.mask['CLM']`

Reply:
Makes sense. Added.

(After this is merged, I'll merge `develop` into `develop-v1.0` so that this change will also be applied there.)